lib/tic_tac_toe/board.ex

```diff
@@ -1,27 +1,87 @@
 defmodule TicTacToe.Board do
   alias TicTacToe.Board
+  alias TicTacToe.BoardInspect
+  alias TicTacToe.BoardUpdate
 
-  defstruct [:board_state, game_over: false]
+  defstruct [:board_state]
 
-  defimpl TicTacToe.BoardInspect do
+  defimpl BoardInspect do
     def get(%Board{board_state: board_state}, position) do
       Enum.at(board_state, position - 1)
     end
+
+    def size(%Board{board_state: board_state}) do
+      number_of_positions = length(board_state)
+      row_length = number_of_positions |> :math.sqrt() |> trunc()
+      {number_of_positions, row_length}
```

Comment:
I'm not loving the use of a tuple here. I'm a little sensitive to this because I'm going through some Python code that uses tuples _everywhere_, but do you even need to return row_length as part of this function? You could make a separate function for row_length.

Reply:
The only reason for the tuple is to reduce the public interface but I agree just returning the value in question might be more clear